### PR TITLE
IJPS-92: Update response for duplicate recordSittingRecords

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-92

### Change description ###
Update response for duplicate recordSittingRecords

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by markdathornehmcts — APPROVED** [review]
LGTM

File: src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature [hunk 1]
```diff
@@ -298,7 +330,15 @@ Feature: F-004 - Scenarios for the POST /recordSittingRecords endpoint
     Then a "negative" response is received with a "400 Bad Request" status code
     And the response contains "message" as "008 could not insert"
     And the response contains "errorRecords[0].errorCode" as "POTENTIAL_DUPLICATE_RECORD"
+    And the response contains "errorRecords[0].am" as "true"
```
Comment (SomeshRA) [inline src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature:333]: Could we check createdByName field as well

File: src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature [hunk 2]
```diff
@@ -298,7 +330,15 @@ Feature: F-004 - Scenarios for the POST /recordSittingRecords endpoint
...
+    And the response contains "errorRecords[0].pm" as "false"
+    And the response contains "errorRecords[0].judgeRoleTypeId" as "Judge"
+    And the response contains "errorRecords[0].judgeRoleTypeName" as "Joe Bloggs"
     And the response contains "errorRecords[1].errorCode" as "INVALID_DUPLICATE_RECORD"
```
Comment (SomeshRA) [inline src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature:337]: Same as above

**Review by SomeshRA — COMMENTED** [review]
LGTM

File: src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature (hunk 1 above)
Comment (marianadpereira) [reply src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature:334]: solved in[6bf0f21](https://github.com/hmcts/jps-judicial-payment-service/pull/85/commits/6bf0f210e5a9b1293b0d45cc13e0cecfae860748)

**Pull request merged**